src/hacbs/components/ApplicationDetails/HacbsApplicationDetails.tsx

```diff
@@ -101,24 +122,56 @@ const HacbsApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ appli
                 if (submitClicked) navigate('/app-studio');
               }),
           },
+          {
+            type: 'section-label',
+            key: 'help',
+            label: 'Help',
+          },
           {
             type: 'separator',
             key: 'separator',
             label: '',
           },
           {
             key: 'application-quickstart',
-            label: 'Getting started with an app',
+            label: 'Getting started with an application',
```

Comment:
The title of the quick start should also be updated to align:
![image](https://user-images.githubusercontent.com/14068621/204348417-50c265ca-928e-4bd8-ab34-aa95c2137dc4.png)